lib/whitehall_importer/integrity_checker.rb

```diff
@@ -48,9 +48,19 @@ def body_text_matches?
       proposed_body_text = proposed_payload.dig(:details, :body)
       publishing_api_body_text = publishing_api_content.dig(:details, :body)
 
+      proposed_body_text = remove_html_elements(proposed_body_text, ".gem-c-attachment-link .gem-c-attachment-link__attribute:nth-of-type(2)")
```

Comment:
Ah yes this rings a bell about what you were saying about a generic class name. This seems reasonable. Might be an idea to put this in a variable eg file_size_selector = ".gem-c-attachment-link .gem-c-attachment-link__attribute:nth-of-type(2)" since we use it twice and it's meaning is quite obscure.

Reply:
Done